reference/matlab-api.md

```diff
@@ -513,6 +513,7 @@ The following tables describe the Matlab functions.
 | node = [wb\_supervisor\_node\_get\_self](supervisor.md#wb_supervisor_node_get_from_def)()                                                                            |
 | node = [wb\_supervisor\_node\_get\_from\_def](supervisor.md#wb_supervisor_node_get_from_def)('def')                                                                  |
 | node = [wb\_supervisor\_node\_get\_from\_id](supervisor.md#wb_supervisor_node_get_from_def)('id')                                                                    |
+| s  = [wb\_supervisor\_node\_get\_def](supervisor.md#wb_supervisor_node_get_from_def)(node)                                                                           |
```

Comment:
Wrong tag: `#wb_supervisor_node_get_from_def` should be `#wb_supervisor_node_get_def`.